apps/examples/testcase/ta_tc/systemio/itc/itc_gpio.c

```diff
@@ -25,7 +25,7 @@
 #include <iotbus_gpio.h>
 #include <iotbus_error.h>
 
-#define GPIO_PIN 41
+#define GPIO_PIN 25
```

Comment:
No need to modify this app change for tc because this will be merged into the RTL8720E_support branch, not the master. But tc should be a common module regardless of the board so that we will modify this logic later. So, when we merge RTL8720E code into the master eventually, we should not add this change. Just FYI.

Reply:
yes we will make it generic change in tc and add board dependent values in config

Reply:
Thank you Mr Chang and Sangam, then we will revert all tc temp changes, and will add related defconfig Macro after the changes you mentioned.